### ISSUE TITLE
Multi Select Portal Link Draw

### ISSUE THREAD

**Issue by Judman** [jonatkins/ingress-intel-total-conversion#773]
Bad title, but in the draw tools or via separate plug in, add the ability to select one portal hold a button, ctrl or shift, select another portal and have a line drawn between those two portals. Or select both portals and click an option in the draw tools to create the line.

This would be very helpful for those doing layered and embedded fields where portals are very close together.

**Comment by herbietown** [comment]
Yes I would like this feature as well. How do I select multiple portals to use the auto-draw feature?
